### DISCUSSION TITLE
The meaning of the x/lr0, x/lr1 and x/lr2 graphs output by weights&biases

### DISCUSSION THREAD

**Discussion by gomsis** [ultralytics/yolov5#7593] [Q&A]
What do the x/lr0, x/lr1 and x/lr2 graphs displayed mean when viewing the yolov5 study results in weightS&biases?
I hope you will be able to provide the information.
![スクリーンショット 2022-04-26 184200](https://user-images.githubusercontent.com/92967049/165271758-8b6c4600-bd92-4ccf-8b71-6c2af46d3ef0.png)

**Comment by glenn-jocher** [comment]
@gomsis 3 different learning rates based on 3 parameter groups here:
https://github.com/ultralytics/yolov5/blob/2611477d2eb5725db84b8c971d102b91e36d27a5/train.py#L153-L162

**Reply by gomsis** [reply]
So this calculates the learning rate.Thank you for telling me.
For example,In the graph in this image, the learning rate is decreasing as the number of steps increases.
What does this mean?
Thank you for your cooperation.

**Reply by glenn-jocher** [reply]
@gomsis YOLOv5 🚀 Learning Rate (LR) [schedulers](https://github.com/ultralytics/yolov5/blob/aa542ce6a65658ff931fee9bbab77c0145c152f0/train.py#L175-L181) follow predefined LR curves for the fixed number of `--epochs` defined at training start (`default=300`), and are designed to fall to a minimum LR on the final epoch for best training results.

<img width="850" alt="Screenshot 2022-04-10 at 11 11 51" src="https://user-images.githubusercontent.com/26833433/162611202-4e0b611f-6cca-4e43-94fc-bbc44e6447f8.png">

**Reply by gomsis** [reply]
Thank you very much.
I understood very well.
